pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -16,6 +16,11 @@ module.exports = {
     nodesLinkOld: '//a[contains(text(), "Nodes")]',
     pmmAgentLocator: '//table//td[contains(text(), "PMM Agent")]',
     serviceIdLocatorPrefix: '//table//tr/td[4][contains(text(),"',
+    deleteButton: '//span[contains(text(), "Delete")]',
+    proceedButton: '//span[contains(text(), "Proceed")]',
+    forceModeCheckbox:
+      '//div[@data-qa="form-field-force"]/descendant::span[@class="checkbox-container__checkmark"]',
+    tableCheckbox: '//div[@data-qa="select-row"]',
```

Comment:
Pls prefer css wherever possible: `div[data-qa="select-row"]`

Reply:
Codecept has a lot of issues with css. Sometimes they don't work, that's why we are back with Xpath. Xpaths are more stable for codecept.

Reply:
this ^ just works in other places, therefore should work here

Reply:
Our decision was to use xpath only when we can't use css, IIRC

Reply:
We had the conversation on June 22nd. Look at the screenshot:
![image](https://user-images.githubusercontent.com/63704168/87286942-c76e0b00-c4f9-11ea-9896-730fdd6fcfdc.png)
Look at the code and see that it's failing. Nobody knows why, but it happening sometimes. :)

Reply:
Looking at the the screenshot both are passing. What am I missing?